### ISSUE TITLE
Can't remove border on Android.

### ISSUE THREAD

**Issue by DiegoPatrocinio** [robinmanuelthiel/flexbutton#32]
Hi there!

After updating to 0.6.1, I can't remove the border on Android anymore, on 0.5.0 worked fine.

Even if I set:

BorderThickness="0" HighlightBorderColor="Transparent" BorderColor="#00FFFFFF"

It still shows a white border, am I doing anything wrong here?

**Comment by DiegoPatrocinio** [comment]
Now even after I roll back to 0.5.0, clean everything and delete the output folders, the whiter border still shows up, can't get rid of it anymore.

**Comment by IeuanWalker** [comment]
@DiegoPatrocinio @juanrobertodeanda  Same here, it happened to my app after I update the Xamarin.forms NuGet to version 3. It might also relate to this issue https://github.com/robinmanuelthiel/flexbutton/issues/44

**Comment by robinmanuelthiel** [comment]
Yep, can confirm that. Looks like a Xamarin.Forms 3.0 thing, correct. They changed a lot on Frames. Some cool things, where we can benefit from but also some strange defaults. Will dig into that!

**Comment by robinmanuelthiel** [comment]
It's an official Xamarin.Forms bug: https://github.com/xamarin/Xamarin.Forms/issues/2634
